backend/src/plugins/offboard/offboard_service_impl.h

```diff
@@ -0,0 +1,116 @@
+#include "plugins/offboard/offboard.h"
+#include "offboard/offboard.grpc.pb.h"
+
+namespace dronecode_sdk {
+namespace backend {
+
+template<typename Offboard = Offboard>
+class OffboardServiceImpl final : public rpc::offboard::OffboardService::Service {
```

Comment:
I think you need the full namespace here:
```
dronecode_sdk::rpc::offboard::OffboardService::Service
```

Reply:
That's weird, it is not required e.g. [here](https://github.com/Dronecode/DronecodeSDK/blob/develop/backend/src/plugins/action/action_service_impl.h#L8), for some reason :thinking:

Reply:
Ok, maybe it works without and the problem was just the `Offboard` vs `offboard`.

Reply:
I would try that first :ok_hand: